### PR TITLE
Execution started event

### PR DESCRIPTION
**Pull request by CDimonaco**
Add the ExecutionStarted event from contracts, published to the `results` queue of rabbitmq.

This event will later by consumed by the web dashboard.

cc @jamie-suse

### CODE REVIEW

File: lib/wanda/messaging/mapper.ex
Comment (arbulu89) [inline lib/wanda/messaging/mapper.ex:25]: `Target` should be a list -> `[Target.t()]`

File: lib/wanda/messaging/mapper.ex
Comment (arbulu89) [inline lib/wanda/messaging/mapper.ex:97]: Strange the conversion from `checks` to `target_checks` hehe
But I guess is a copy/paste from somewhere else

File: mix.exs
Comment (arbulu89) [inline mix.exs:112]: I guess this is temporal until we merge the contract PR, right?

**Review by arbulu89 — COMMENTED** [review]
Nice! I would wait until we merge the contracts repo to put the commit of the merge.
Besides that, all fine

File: mix.exs
Comment (CDimonaco) [reply mix.exs:112]: yes!

File: lib/wanda/messaging/mapper.ex
Comment (CDimonaco) [reply lib/wanda/messaging/mapper.ex:97]: 👀  you're right

File: test/wanda/messaging/mapper_test.exs
Comment (fabriziosestito) [inline test/wanda/messaging/mapper_test.exs:34]: let's alias these

**Review by fabriziosestito — APPROVED** [review]
LGTM, :shipit:  when the linter is happy + nitpick

File: mix.exs
Comment (arbulu89) [reply mix.exs:112]: Man, at the end you didn't wait until merging first the other to put here the merge commit XD
Now this commit doesn't even exist in the `main` branch, which looks difficult to debug

**Pull request merged**
